### DISCUSSION TITLE
Does Zod support labels?

### DISCUSSION THREAD

**Discussion by vedantroy** [airjp73/rvf#106] [Q&A] [answered]
This is maybe more of a zod question than a remix-validated-form, but I was wondering if zod supports labels, like yup. Right now, my error messages say: "string must be at least 5 character(s)", which is pretty generic.

I ctrl-f'd through the zod readme but I didn't find anything.

**Comment by airjp73** [comment, marked as answer]
There's no label helper in zod, but you can customize error messages. Here's some docs: https://github.com/colinhacks/zod/blob/v3/ERROR_HANDLING.md#customizing-errors-with-zoderrormap
